pkgs/tools/networking/corerad/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "corerad";
+  version = "0.1.1";
+
+  goPackagePath = "github.com/mdlayher/corerad";
+
+  src = fetchFromGitHub {
+    owner = "mdlayher";
+    repo = "corerad";
+    rev = "v${version}";
+    sha256 = "02zx8h298x4c5zgj25l8l9m8a2ygj2bafvzh7hj7c6qifir7pamb";
+  };
+
+  modSha256 = "036b0cqj9ssmcjdvrxvibzchdc3cyxbj868fyv5z81b53xlqn3np";
```

Comment:
```suggestion
  modSha256 = "0cfjsiiiff0k1v5j837s4jbp02snqrmv08grm69b3wjhr4145886";
```

Reply:
Thanks. On my own 19.09 stable machine with the existing hash, I'm seeing:

```
[matt@routnerr-2:~/src/nixpkgs]$ nix-build -A corerad
/nix/store/1p9mx227m0yr8y9di48vca3fs6hbylar-corerad-0.1.1

[matt@routnerr-2:~/src/nixpkgs]$ ./result/bin/corerad -h
Usage of ./result/bin/corerad:
  -c string
        path to configuration file (default "corerad.toml")
  -init
        write out a default configuration file to "corerad.toml" and exit
```

I'm new to Nix and NixOS and am not sure what all is used to compute this hash. Would you mind sharing what you're doing?

Reply:
Interestingly, your hash also seems to work. I wonder if I got a cache into a bad state somehow. Will apply this, thanks.

```
[matt@routnerr-2:~/src/nixpkgs]$ git diff
diff --git a/pkgs/tools/networking/corerad/default.nix b/pkgs/tools/networking/corerad/default.nix
index beac60dbb96..992f5c4365a 100644
--- a/pkgs/tools/networking/corerad/default.nix
+++ b/pkgs/tools/networking/corerad/default.nix
@@ -13,7 +13,7 @@ buildGoModule rec {
     sha256 = "02zx8h298x4c5zgj25l8l9m8a2ygj2bafvzh7hj7c6qifir7pamb";
   };
 
-  modSha256 = "036b0cqj9ssmcjdvrxvibzchdc3cyxbj868fyv5z81b53xlqn3np";
+  modSha256 = "0cfjsiiiff0k1v5j837s4jbp02snqrmv08grm69b3wjhr4145886";
 
   meta = with stdenv.lib; {
     homepage = "https://github.com/mdlayher/corerad";

[matt@routnerr-2:~/src/nixpkgs]$ nix-build -A corerad
/nix/store/v9ccf7jf95da4vfv83prbm046b938lf4-corerad-0.1.1

[matt@routnerr-2:~/src/nixpkgs]$ ./result/bin/corerad -h
Usage of ./result/bin/corerad:
  -c string
        path to configuration file (default "corerad.toml")
  -init
        write out a default configuration file to "corerad.toml" and exit
```